### DISCUSSION TITLE
The performance of the DPA2 model is an order of magnitude worse than that of DPA1

### DISCUSSION THREAD

**Discussion by duduRUHappy** [deepmodeling/deepmd-kit#3915] [Q&A]
When I used DPA1 for testing on a new dataset, the results were as follows: rmse_energy_valid = 0.0361 eV and rmse_force_valid = 0.29 eV/Å. These results are in line with expectations. However, when I used the single task model version of  DPA2 for training and testing, the results were rmse_energy_valid = 0.1361 eV and rmse_force_valid = 1.8297 eV/Å. As the results indicate, the performance of the DPA2 model is an order of magnitude worse than that of DPA1. Could you help analyze how to improve the DPA2 results?

I adopted an input.json similar to the one published for the DPA2 model. The details of input.json of DPA1 and DPA2 are included in the attachment, please take a look.
[dpa1_ener_input.json](https://github.com/user-attachments/files/16009052/dpa1_ener_input.json)
[dpa2_st_ener_input.json](https://github.com/user-attachments/files/16009056/dpa2_st_ener_input.json)

**Comment by wanghan-iapcm** [comment]
from the input script it seems that you are using a very old version of the code. please report the version, or you may want to try the latest devel branch or the 2024Q1 stable branch.

**Reply by duduRUHappy** [reply]
Yes, I used the 'devel' branch from git@github.com:dptech-corp/deepmd-pytorch.git.

**Reply by duduRUHappy** [reply]
![image](https://github.com/deepmodeling/deepmd-kit/assets/49438740/0bc27992-18c5-448e-b566-146a577bd679)

**Reply by wanghan-iapcm** [reply]
the deepmd-pytorch code is no longer supported

**Reply by duduRUHappy** [reply]
Ok, I got it. Then if I wanna test DPA2, should I use [v3.0.0a0](https://github.com/deepmodeling/deepmd-kit/releases/tag/v3.0.0a0) or the latest [v2.2.10](https://github.com/deepmodeling/deepmd-kit/releases/tag/v2.2.10)?

**Reply by njzjz** [reply]
It's not in v2. Perhaps a deprecation notice should be added to `deepmd-pytorch`. cc @iProzd
